File: src/androidTest/java/pt/joaomneto/titancompanion/TestDD.java
package pt.joaomneto.titancompanion;


import android.support.test.filters.LargeTest;
import android.support.test.runner.AndroidJUnit4;

import org.junit.Test;
import org.junit.runner.RunWith;

import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook;

import static pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.DEATHTRAP_DUNGEON;
import static pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.THE_FOREST_OF_DOOM;
import static pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.THE_WARLOCK_OF_FIRETOP_MOUNTAIN;

@LargeTest
@RunWith(AndroidJUnit4.class)
public class TestDD extends TestTFOD{

    @Override
    protected FightingFantasyGamebook getGamebook(){
        return DEATHTRAP_DUNGEON;
    }

}

File: src/androidTest/java/pt/joaomneto/titancompanion/phase1/TestDD.kt
package pt.joaomneto.titancompanion.phase1

import androidx.test.filters.LargeTest
import androidx.test.ext.junit.runners.AndroidJUnit4
import org.junit.runner.RunWith
import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.DEATHTRAP_DUNGEON

@LargeTest
@RunWith(AndroidJUnit4::class)
class TestDD : TestTFOD() {

    override val gamebook = DEATHTRAP_DUNGEON
}
